_data/navigation/en_US.yml

```diff
@@ -79,9 +79,6 @@ sidebar_pages:
   -
     title: Flashing ntrboot (NDS)
     url: flashing-ntrboot-(nds)
-  -
-    title: Homebrew Launcher (PicHaxx)
-    url: homebrew-launcher-(pichaxx)
```

Comment:
me when i remove a page we're actively using

Reply:
I can't read.

No better than some of our users...